optimum/exporters/onnx/convert.py

```diff
@@ -61,6 +61,66 @@ def check_dummy_inputs_are_allowed(
         )
 
 
+def validate_encoder_decoder_model_outputs(
+    config: OnnxConfig,
+    reference_model: Union["PreTrainedModel", "TFPreTrainedModel"],
+    onnx_named_outputs: List[str],
+    atol: float,
+    task: str,
+    use_past: bool,
+    encoder_onnx_model: Path,
+    decoder_onnx_model: Path,
+    decoder_with_past_onnx_model: Path = None,
+):
+    """
+    Validates the export by checking that the outputs from both the reference and the exported model match.
+    The following method validates the ONNX models exported using the `export_encoder_decoder_model` method.
+
+    Args:
+        config ([`~OnnxConfig`]:
+            The configuration used to export the model.
+        reference_model ([`~PreTrainedModel`] or [`~TFPreTrainedModel`]):
+            The model used for the export.
+        onnx_named_outputs (`List[str]`):
+            The names of the outputs to check.
+        atol (`float`):
+            The absolute tolerance in terms of outputs difference between the reference and the exported model.
+        task (`str`)
+            The type of task to export the model with.
+        use_past (`bool`, *optional*, defaults to `None`):
```

Comment:
```suggestion
        use_past (`bool`):
```

Reply:
Is it needed since we both have the `task` and the `config` arguments?

Reply:
Removed.